### DISCUSSION TITLE
Add Slider for How Fast to Fade Out UR Bar Information

### DISCUSSION THREAD

**Discussion by cesarila2** [ppy/osu#27531] [Ideas]
First of all, I really appreciate the customizability of the UR bar compared to stable. I find it much more useful in lazer. That being said, it pales in comparison to other VSRGs I've played. The most useful thing to add (in my opinion) is the ability to be able to tell the hit feedback to fade out much faster.

My first thought of how to do this would be to add a slider for how long to persist each thing on the UR. Bonus points for adding another slider for the minimum amount of feedback to keep on the display, e.g. always show feedback for the last 5 objects where possible.

Example image of how I'd want to interact with this:
![image](https://github.com/ppy/osu/assets/89962320/468abd46-bfa8-48b3-b564-fd6351a4de98)

While I'm at it, I may as well list as many things as possible to improve the UR bar for mania (still, in my opinion):

- Add UR bar color for 320s in mania. 300s cost us accuracy now, so it feels more useful than in stable. I'll suggest purple, but whatever color is fine.
- Add an option to change the colors of each UR bar region (not at all necessary but would be cool)

EDIT: more -> most (typo)
EDIT2: Move additional wish list items from separate comment to this one because the comment closed the discussion.
EDIT3: Sorry for closing and opening this issue a couple times. It was not obvious to me how the close/open discussion feature wasn't tied to making comments. I believe this should be the last time I need to edit this. Thanks for your patience.

**Comment by ItsCollector** [comment]
i'm a mania player as well and i would like this implemented too. i was about to make a post requesting this feature.

**Comment by peppy** [comment]
Wouldn't be against these changes being added 👍
